### DISCUSSION TITLE
geometrical coherence in Network.py

### DISCUSSION THREAD

**Discussion by rainxinyucao** [isce-framework/fringe#78] [Q&A]
Hello, what is the physical meaning of geometrical coherence in Network.py? Unable to download relevant literature, do not quite understand the meaning of this variable。
Thanks.

**Comment by scottstanie** [comment]
Hi, here is the most common reference talking about that: 

[Decorrelation in interferometric radar echoes](https://ieeexplore.ieee.org/abstract/document/175330/)
HA Zebker, J Villasenor

The relevant equation:

![image](https://user-images.githubusercontent.com/8291800/206511661-4bb08c58-fdfd-4195-99c3-c70d62561250.png)

This is decorrelation coming from the separation of the two satellites (hence the "geometry" of the interferogram network).

- $R_y$ is the range resolution (or range spacing below)
- B is the spatial baseline
- $\theta$ is the incidence angle 
`corr = 1.0 - 2*B*rangeSpacing*((np.cos(theta))**2)/wvl/r`

**Reply by rainxinyucao** [reply]
I am very happy to receive your reply. I have been working on the calculation of some baseline coefficients through code recently, and your answer is very helpful to me.Thanks！
